mysql-test/suite/rpl/t/rpl_bug85158.test

```diff
@@ -0,0 +1,108 @@
+#
+# Testcase for bug 85158.
+#
+--source include/not_group_replication_plugin.inc
+--source include/have_gtid.inc
+--source include/have_binlog_format_row.inc
+
+--let $rpl_multi_source= 1
+--let $rpl_topology= 1->2,2->3,1->3
+--source include/rpl_init.inc
+
+# mi writes threshold, if the difference between count of writes to
```

Comment:
s/mi/MI/g (it's an abbreviation)

Reply:
between the count to the MI repository (two "the")

Reply:
fixed